apps/console/src/features/claims/components/wizard/external-dialect/external-claims-add-dialect.tsx

```diff
@@ -106,15 +107,37 @@ export const ExternalClaims: FunctionComponent<ExternalClaimsPropsInterface> = (
                                     isLoading={ false }
                                     list={ claims }
                                     localClaim={ ListType.ADD_EXTERNAL }
-                                    onEdit={ (index: number, values: Map<string, FormValue>) => {
+                                    onEdit={ (editingClaim: Claim | ExternalClaim | ClaimDialect | AddExternalClaim,
+                                              values: Map<string, FormValue>) => {
+
                                         const tempClaims = [ ...claims ];
-                                        tempClaims[ index ].mappedLocalClaimURI = values.get("localClaim").toString();
-                                        tempClaims[ index ].claimURI = values.get("claimURI").toString();
+
+                                        tempClaims.forEach((claim: AddExternalClaim) => {
+                                            if (!("claimURI" in editingClaim)) {
```

Comment:
Shall we get the constant to a constants file?

Reply:
Fixed with `#0f64603`